Cargo.toml

```diff
@@ -20,13 +20,13 @@ serde_json = "1.0"
 url = "2.5.4"
 kanal = "0.1.0-pre8"
 clap = { version = "4.5.27", features = ["derive"] }
-tokio = "1.43.0"
+tokio = { version = "1.43.0", features = ["rt-multi-thread", "macros"] }
```

Comment:
required to get the multithread aware runtime, a bit curious that it's not a default